lib/msal-common/src/telemetry/TelemetryManager.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { SERVER_TELEM_CONSTANTS, HeaderNames, CacheSchemaType } from "../utils/Constants";
+import { CacheManager } from "../cache/CacheManager";
+import { RequestFailures } from "./RequestFailures";
+import { AuthError } from "../error/AuthError";
+
+export class TelemetryManager {
```

Comment:
jasonnutter Member Pending
TelemetryManager is teh name of the class in v1 that handle telemetry that was passed back to client applications, so I think we need to chat and come up with more distinct names for both.

Reply:
Decided on `ServerTelemetryManager` and `ClientTelemetryManager`

Reply:
Also, can you put these files into a `telemetry/server` folder?

Reply:
Done